microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -683,6 +739,24 @@ All synced data is encrypted in transit over HTTPS when transferred between the
 
 InPrivate and Guest mode browsing data does not sync to your Microsoft account.  However, favorites created during InPrivate sessions are synced across your signed-in versions of Microsoft Edge.
 
+<!-- ====================================================================== -->
+## Tab organization
+
+Microsoft Edge can offer helpful suggestions on how to organize tabs to save you time and keep you focused on the web content you care about. These suggestions augment the built in Tab Grouping feature built into Microsoft Edge.  
+
+When two or more tabs are grouped together, Microsoft Edge sends information about the tabs including the Title and URL to the service over HTTPS.  This information is used to generate a relevant name for the Tab Group.
+
+![TabGroupUI](./media/tabgrouping1.gif)
+
+Additionally, you can choose to have Microsoft Edge auto-group all of your tabs using the “Group Similar Tabs” feature from the Tab Action Menu.  In addition to the Title and URL, Microsoft Edge will also send information on which tabs opened another, and a timestamp on when a tab was opened.  This additional data allows the service to more accurately suggest Tab Groups to organize your tabs. 
+
+![AutoTabGroupUI](./media/tabgrouping2.gif)
```

Comment:
```suggestion
![Auto Tab Group UI](./media/tabgrouping2.gif)
```
gh rendered section:
https://github.com/MicrosoftDocs/edge-developer/blob/user/majenny/ComposePrivacy/microsoft-edge/privacy-whitepaper/index.md#tab-organization

Reply:
confirmed fixed.  also, in my local branch for my PR, fixed tabgroupui above.